.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ioLibrary_Driver"]
+	path = ethernet/w5500_spi/ioLibrary_Driver
+	url = https://github.com/Wiznet/ioLibrary_Driver
```

Comment:
I'm not sure we want to have this library as an actual submodule of pico-examples.

Reply:
yeah i'm fine with that as most people will likely pull in code via a submodule. it is the catch 22 of the examples: how we'd probably ideally do something there is not necessarily a good example of how the user would do it in their standalone project.

So yeah, I think i'm fine with this (if you don't initialize the submodule you don't get the example).... this is of course a minor doc issue

Reply:
>  …this is of course a minor doc issue

Yeah. I'm going to have to think how to add a level of subtlety into the documentation around that.

Reply:
I messed it up in the previous commit but have updated it since; the files shouldn't have copied over to pico-examples too. @kilograham said it would be best to add it in as a submodule. And it would stay sync with any library updates